process/github-policy.md

```diff
@@ -0,0 +1,82 @@
+Open Mainframe Project hosted projects generally use GitHub for code hosting and issue management. GitHub organizations for hosted projects are owned and admistered by the Linux Foundation staff, including the Linux Foundation release engineering team, to ensure the sustainability of the infrastructure. 
+
+This document outlines the policies and procedures for projects using GitHub for code hosting. In addition, the Linux Foundation release engineering team maintains [documentation](https://docs.releng.linuxfoundation.org/en/latest/) on it's services, policies, and procedures.
+
+## New project or repository
+
+When a new project or repository is to be added, please [submit a request to the Open Mainframe Project staff](https://github.com/openmainframeproject/foundation/issues/new/choose) who can faciliate the process.
+
+### Code license scan
+
+If adding the new project or repository will include a code dump, it's highly recommended to request a code license scan before bringing the code in. This scan will be a validation for...
+
+- All files include correct license headers and copyright statements ( refer to the [License Specification in the Contribution Guidelines](https://github.com/openmainframeproject/tac/blob/master/process/contribution_guidelines.md#license-specification) for more information ).
+- There isn't any incompatible licensed code ( either with a non-OSI license or a license that is incompatible with the project's license ).
+- Any other best practices guidance.
+
+Typically code license scans are a fairly quick turnaround, but that might take longer for bigger code bases.
+
+### Repository setup
+
+Generally most projects coming into the Open Mainframe Project utilize the existing Open Mainframe Project GitHub organization, especially if they intend to have a single repository. This practice enables the easiest discoverablity of the project.
+
+If a project anticipates having multiple repositories, the Open Mainframe Project staff can provision a GitHub organization specific to that project. With either option, the same policies apply for adminstration and access control.
+
+### Code migration
+
+There are two strategies for migrating the code to an Open Mainframe Project managed repository.
+
+- Best practice is for the Open Mainframe Project staff to provision a new repository, where the new code can be contributed to as a pull request. 
+- If preserving the commit history is important, then the repository can be transferred using the [GitHub transfer repository](https://docs.github.com/en/github/administering-a-repository/transferring-a-repository) process. If doing this, then before the transfer you must add a commit to the repository providing a DCO signoff for all previous commits. Easiest way is to check in a file called `past-dco-signoff.txt` with the contents below...
+
+```
+I, <AUTHOR NAME> hereby sign-off-by all of the past commits to this repo subject to the Developer Certificate of Origin (DCO), Version 1.1. 
```

Comment:
Typically I would suggest include both name and email address for this sign-off, since regular DCO sign-offs usually include name and email address.

I might also change it to specify something like "all of the commits prior to and including <COMMIT_HASH>" since after the file is checked into the repo, it won't subsequently be clear which commit it was referring to (without digging into the git history).